### PR TITLE
Fix styles and instructions in docs

### PR DESCRIPTION
**Pull request by calvinxiao**
1. Make command to create MySQL user consistent across docs and init.sql
2. Remove markdown syntax in code blocks
3. Fix some markdown style problems

**Why?**
Improve docs quality

**How did you test it?**
No tests required

**Potential risks**
No

**Release notes**

**Documentation Changes**

### CODE REVIEW

File: docs/persistence.md [hunk 1]
```diff
@@ -40,9 +40,8 @@ When run tests and CLI command locally, Cadence by default uses a user `uber` wi
 You can use the following command to create user(role) and grant access. 
 In the mysql shell:
 ```
-> CREATE USER 'uber' IDENTIFIED BY 'uber';
-> GRANT All Privileges to 'uber';
-
+> CREATE USER 'uber'@'%' IDENTIFIED BY 'uber';
+> GRANT ALL PRIVILEGES ON *.* TO 'uber'@'%';
```
Comment (longquanzheng) [inline docs/persistence.md:44]: It should be fine but just want to check, do you know what's the differences? It seems like All is more than *.*

Comment (longquanzheng) [reply docs/persistence.md:44]: And also what difference it makes for `'uber'@'%'`?

**Comment by longquanzheng** [comment]
Just two comments about information. The PR LGTM. Thanks for fixing them!

File: docs/persistence.md (hunk 1 above)
Comment (calvinxiao) [reply docs/persistence.md:44]: 1. `GRANT ALL PRIVILEGES TO 'uber'@'%';` without `ON *.*` doesn't work on MySQL8.0 via docker, I know I should've used 5.7. For people didn't read all the full documentation, `docker pull mysql` will pull `8.*` version.
2. `The host name part of the account name, if omitted, defaults to '%'.`, I just copied them from [/docs/setup/MYSQL_SETUP.md](/docs/setup/MYSQL_SETUP.md) and [/docker/buildkite/mysql-init/init.sql](/docker/buildkite/mysql-init/init.sql), so they look the same.

**Review by longquanzheng — APPROVED** [review]

**Pull request merged**